recipes/libcap/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+import os
+import glob
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibcapConan(ConanFile):
+    name = "libcap"
+    version = "2.45"
```

Comment:
```suggestion
```

Don't hard code the version.

You can provide locally the version number via CLI. For example `conan create . libcap/2.45@`

Reply:
Thanks. I just forgot to delete `version` after `conan new`.